presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/TestCassandraDistributedQueries.java

```diff
@@ -160,6 +160,12 @@ protected TestTable createTableWithDefaultColumns()
         throw new SkipException("Cassandra connector does not support column default values");
     }
 
+    @Override
+    public void testColumnName(String columnName)
+    {
+        // Cassandra connector currently does not support create table
```

Comment:
does it support CTAS?

Reply:
Yes, CTAS is already supported.

Reply:
let's cover this in https://github.com/prestosql/presto/pull/3478 maybe.

Reply:
i removed the override, tests will pass when #3478 is merged

Reply:
I added the override back. The test fails for cassandra for a reason i don't understand.
let's get the test in in its general shape so that we can collaborate on it